themes/scss/map/_overlays.scss

```diff
@@ -303,85 +316,86 @@ $maxLegendContainerHeight: 300px;
 }
 
 .Bubble-list {
-  @include display-flex();
-  @include justify-content(center);
-  position: relative;
-  width: 100px;
+  width: 100%;
   height: 100px;
 }
-.Bubble-list::before {
-  position: absolute;
-  right: 50%;
-  bottom: 0;
-  width: 140px;
-  height: 1px;
-  background: rgba(0, 0, 0, 0.08);
-  content: '';
-  visibility: visible;
-}
+
 .Bubble-item {
-  @include opacity(0.5);
-  @include transform(translateX(-50%));
   position: absolute;
-  bottom: 0;
-  left: 50%;
+  width: 100%;
+  height: 100px;
 }
 
 .Bubble-container {
-  @include display-flex();
-  max-height: 100px;
-  padding-top: 3px;
-  padding-bottom: 8px;
+  position: relative;
+  top: 7px;
+  right: 0;
+  width: 100%;
+  height: 100px;
+  margin-top: 10px;
 }
+
+.Bubble-circle {
+  @include opacity(0.5);
+  position: absolute;
+  right: 0;
+  width: 100px;
+  height: 100%;
+}
+
 .Bubble-itemCircle {
   display: block;
-  position: relative;
+  position: absolute;
+  bottom: 0;
+  left: 50%;
   width: 100%;
   height: 100%;
+  transform: translate3d(-50%, 0, 0);
   border: 1px solid rgba(0, 0, 0, 0.1);
   border-radius: 50%;
   background: #F8AB17;
   box-shadow: -1px 0 2px 0 rgba(0, 0, 0, 0.1);
   pointer-events: none;
 }
-.Bubble-item::before {
+
+.Bubble-label {
   position: absolute;
-  top: 5px;
-  right: 50%;
-  width: 140px;
-  height: 1px;
-  background: rgba(0, 0, 0, 0.08);
-  content: '';
+  bottom: 0;
+  width: calc(100% - 50px);
+
+  &::after {
+    position: absolute;
+    top: -1px;
+    left: 0;
+    width: 100%;
+    height: 1px;
+    background: rgba(0, 0, 0, 0.08);
```

Comment:
I prefer `#000` in these cases

Reply:
The thing is that we cannot use `#000` because it isn't its original color.

With `#000`:
![screen shot 2018-01-29 at 16 56 57](https://user-images.githubusercontent.com/4319728/35519858-b693c78a-0515-11e8-975d-222c899ba3b4.png)

And with `rgba(0, 0, 0, 0.08)` (as it should be):
![screen shot 2018-01-29 at 16 59 34](https://user-images.githubusercontent.com/4319728/35519898-d72e415a-0515-11e8-9a32-079eb25dd968.png)

Reply:
yeah, I mean `rgba(#000, 0.08)` should be good